crates/ty_project/src/metadata/options.rs

```diff
@@ -296,14 +297,22 @@ impl Options {
         };
 
         // collect the existing site packages
-        let mut extra_paths: Vec<SystemPathBuf> = Vec::new();
+        let mut extra_paths: Vec<SystemPathBuf> = environment
+            .extra_paths
+            .as_deref()
+            .unwrap_or_default()
+            .iter()
+            .map(|path| path.absolute(project_root, system))
+            .collect();
 
         // read all the paths off the PYTHONPATH environment variable, check
         // they exist as a directory, and add them to the vec of extra_paths
         // as they should be checked before site-packages just like python
         // interpreter does
-        if let Ok(python_path) = system.env_var("PYTHONPATH") {
-            for path in python_path.split(':') {
+        if let Ok(python_path) = system.env_var(EnvVars::PYTHONPATH) {
+            const SEP: &str = if cfg!(windows) { ";" } else { ":" };
```

Comment:
I think we may still want to use `split_paths` here for:

> This also performs unquoting on Windows.

Reply:
Ah I didn't see that previously.